src/webview/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -44,8 +44,11 @@
 	private static final int NON_SIMPRINTS_FLAGS = 0x7;
 	static final int GRAB_PHOTO = (0 << 3) | NON_SIMPRINTS_FLAGS;
 	static final int GRAB_MRDT_PHOTO = (1 << 3) | NON_SIMPRINTS_FLAGS;
+	private static final int ACCESS_FINE_LOCATION_PERMISSION_REQUEST = (int)(Math.random() * 1000);
 
-	private final static int ACCESS_FINE_LOCATION_PERMISSION_REQUEST = (int)(Math.random() * 1000);
+	private static final int DISCLOSURE_LOCATION_PERMISSION_REQUEST = 1122331;
```

Comment:
I think this should be random as above.

Reply:
Yes same than above, I'll change it but I would like to understand what we do that.